unit-tests/live/rec-play/test-got-playback-frames.py

```diff
@@ -179,7 +193,7 @@ def stop_sensor( sensor ):
 
     # if record and playback worked we will receive frames, the callback functions will be called and got-frames
     # will be True. If the record and playback failed it will be false
-    test.check( got_frames )
+    test.check( got_frames_depth and got_frames_rgb )
```

Comment:
I think it's better to split into 2 checks..
If we get logs we can identify to which sensor the issue relates to.

Reply:
Agree, I separated it